mix.exs

```diff
@@ -21,9 +21,9 @@ defmodule Basic.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:membrane_core, "~> 0.8.0"},
-      {:membrane_caps_video_raw, "~> 0.1.0"},
-      {:membrane_caps_audio_raw, "~> 0.5.0"},
+      {:membrane_core, "~> 0.12.0"},
+      {:membrane_raw_video_format, "~> 0.3.0"},
+      {:membrane_raw_audio_format, "~> 0.11.0"},
       {:mock, "~> 0.3.0", only: :test}
       # {:dep_from_hexpm, "~> 0.3.0"},
       # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
```

Comment:
Same as in case of the previous PR - I know that it is out of scope of that PR, but we can remove that ;)